lib/js/components/Pill/Pill.vue

```diff
@@ -87,6 +132,19 @@ $pill-colors: (
 			#{$self}__label {
 				color: map-get($color-map, 'label');
 			}
+
+			&:disabled,
```

Comment:
I don't think this makes sense outside of form controls: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled